.gitattributes

```diff
@@ -6,3 +6,4 @@
 *.pkl filter=lfs diff=lfs merge=lfs -text
 *.tar.gz filter=lfs diff=lfs merge=lfs -text
 *.tgz filter=lfs diff=lfs merge=lfs -text
+*.tar.xz filter=lfs diff=lfs merge=lfs -text
```

Comment:
so far, all the other models are tar.gz. for consistency, would be better to keep the same. any particular reason to use tar.xz?

Reply:
No. I just uploaded tar.gz and updated the PR